### ISSUE TITLE
Text style undone when de-selecting all

### ISSUE THREAD

**Issue by JcdCoder** [KillerCodeMonkey/ngx-quill#328]
I've noticed an issue in these browsers:
- Chrome Version 71.0.3578.98 (Official Build) (64-bit) Desktop
- Chrome Version 71.0.3578.99 Android
- Safari Version 12.0 (14606.1.36.1.9)

**Steps to reproduce the issue:**

1. In editor type some text (e.g., "bold text").
2. Select all text.  
3. Click the bold button in the toolbar.  RESULT: all text is bold.
4. Click in text area to deselect text.

**ACTUAL RESULT:** Text is no longer bold.
**EXPECTED RESULT:** Text is still bold.

**Notes:** 
- This behavior also happens with styles other than bold (e.g., underline, italics, bullets, etc.)
- This behavior does not occur when a substring is selected in step 2.  You must select all text to reproduce this issue.
- I am unable to reproduce this issue using Firefox 64.0
- I am unable to reproduce this issue using the iOS Simulator (with Safari).

I'm not sure whether this is quilljs or ngx-quill behavior.  

Thanks for any help you can provide!

**Comment by KillerCodeMonkey** [comment]
And btw. I can not Reproduce this issue on mobile Chrome 71.03578.99

I opened my Demo and typed some Text in the First Editor. Selecting all,
chose bold and Clicked in the textarea again. Everthing stays AS expected

Am Di., 5. Feb. 2019, 20:28 hat Bengt Weiße <bengtler@googlemail.com>
geschrieben:

**Comment by KillerCodeMonkey** [comment]
So tested in on my laptop in chrome with v71.0.3578.98.
No such problems. There are quilljs problems (like in other wysiwyg editors) with plugins like grammarly or IME addons or IME software on your pc

**Comment by JcdCoder** [comment]
Thanks for the quick replies!

I have noticed issues with grammarly in the past, so I have already disabled it for domains in which I'm using ngx-quill.  

Using the same (problematic) Chrome browser, I tried to reproduce the issue in the quilljs playground and the ngx-quill demo.  I am not able to reproduce the issue in either of these.  

Here's what I've noticed.  This issue is only present when the name attribute is defined on <quill-editor>.

**Issue Exists**
`<quill-editor name="test"></quill-editor>`

**No Issue Here**
`<quill-editor></quill-editor>`
